src/Xrefcheck/CLI.hs

```diff
@@ -97,6 +98,9 @@ defaultConfigPaths = ["./xrefcheck.yaml", "./.xrefcheck.yaml"]
 -- and flavors, so we write a type alias here.
 type RepoType = Flavor
 
+filepathOption :: Mod OptionFields FilePath -> Parser FilePath
+filepathOption = fmap normalise <$> strOption
```

Comment:
We should probably do this in _all_ places where we read filepaths from an external source / use hardcoded filepaths.

As far as I can tell, we're only missing 2 places: `readConfig` and `defConfig`. We should probably have a `normaliseFilePaths :: Config -> Config` function, and use it inside `readConfig` / `defConfig`.